templates/zerver/integrations/github-actions.md

```diff
@@ -1,8 +1,25 @@
-It's easy to send Zulip messages from GitHub Actions! This is useful:
+# Zulip GitHub Actions integration
 
-  * If you want to report the success or failure of an Action to Zulip.
-  * If you want to send output from your actions to Zulip.
+It's easy to send Zulip messages from GitHub Actions!
 
-Instructions are available [here](https://github.com/zulip/github-actions-zulip#readme).
+{start_tabs}
 
-See also the [GitHub integration](/integrations/doc/github).
+1. {!create-a-generic-bot.md!}
```

Comment:
Removed the "Note down the bot's email address and API key." line that was appended to this instruction.